src/EnergyPlus/MixedAir.cc

```diff
@@ -1566,100 +1556,105 @@ void GetOAControllerInputs(EnergyPlusData &state)
-                    thisVentMechZone.ZoneOAPeopleRate = 0.00944;
-                    thisVentMechZone.ZoneOAFlowRate = 0.0;
-                    thisVentMechZone.ZoneOAACHRate = 0.0;
-                    thisVentMechZone.ZoneOAFlowMethod = OAFlowCalcMethod::PerPerson;
-                    thisVentMechZone.zoneOASched =
-                        Sched::GetScheduleAlwaysOn(state); // defaults to constant-1.0. TODO: what is this really suppoed to be?
+                Real64 zoneOAAreaRate = 0.0;
+                Real64 zoneOAPeopleRate = 0.0;
+                if (thisVentMechZone.ZoneDesignSpecOAObjIndex == 0) {
+                    // use defaults
+                    thisVentMechZone.ZoneDesignSpecOAObjIndex = DataSizing::getDefaultOAReq(state);
                     ShowWarningError(state, format("{}{}=\"{}", RoutineName, CurrentModuleObject, thisVentilationMechanical.Name));
                     ShowContinueError(
                         state, format("Cannot locate a matching DesignSpecification:OutdoorAir object for Zone=\"{}\".", thisVentMechZone.name));
                     ShowContinueError(state, "Using default OA of 0.00944 m3/s-person and 0.0 m3/s-m2.");
                 }
-                int zoneAirDistObjIndex = thisVentMechZone.ZoneDesignSpecADObjIndex;
-                if (zoneAirDistObjIndex > 0) {
-                    auto const &curZoneAirDistribution(state.dataSize->ZoneAirDistribution(zoneAirDistObjIndex));
-                    thisVentMechZone.ZoneADEffCooling = curZoneAirDistribution.ZoneADEffCooling;
-                    thisVentMechZone.ZoneADEffHeating = curZoneAirDistribution.ZoneADEffHeating;
-                    thisVentMechZone.zoneADEffSched = curZoneAirDistribution.zoneADEffSched;
-                    thisVentMechZone.ZoneSecondaryRecirculation = curZoneAirDistribution.ZoneSecondaryRecirculation;
-                } else { // use defaults
-                    thisVentMechZone.ZoneADEffCooling = 1.0;
-                    thisVentMechZone.ZoneADEffHeating = 1.0;
-                    thisVentMechZone.ZoneSecondaryRecirculation = 0.0;
-                    ShowWarningError(state, format("{}{}=\"{}\"", RoutineName, CurrentModuleObject, thisVentilationMechanical.Name));
-                    ShowContinueError(
-                        state,
-                        format("Cannot locate a matching DesignSpecification:ZoneAirDistribution object for Zone=\"{}\".", thisVentMechZone.name));
-                    ShowContinueError(state, "Using default zone air distribution effectiveness of 1.0 for heating and cooling.");
+                assert(thisVentMechZone.ZoneDesignSpecOAObjIndex > 0);
+                auto &curOARequirements = state.dataSize->OARequirements(thisVentMechZone.ZoneDesignSpecOAObjIndex);
+                zoneOAAreaRate = curOARequirements.desFlowPerZoneArea(state, thisVentMechZone.zoneNum);
+                zoneOAPeopleRate = curOARequirements.desFlowPerZonePerson(state, thisVentMechZone.zoneNum);
```

Comment:
Can't you just declare these Real64 here and delete 1559 and 1560? Cppcheck will tag those declarations with "assigned a value that is never used" (I think).

Reply:
> Can't you just declare these Real64 here and delete 1559 and 1560? Cppcheck will tag those declarations with "assigned a value that is never used" (I think).

Yes, thanks. Leftover from an earlier iteration on this. Removed.